packages/ui/src/IntervalCard/NotesSection.tsx

```diff
@@ -55,10 +55,10 @@ const NotesSection: React.FC<NotesSectionProps> = ({
       <Form className={className}>
         <FastField
           name="bookingNotes"
-          className="w-full h-full min-h-[105px] border-t border-gray-300"
+          className="w-full h-full min-h-[105px] border-t border-gray-300 "
```

Comment:
Nitpick. Unneeded space before closing quotes. Can be changed in a future PR, even unrelated, or left as is: it won't hurt after all.